packages/sdk-react/src/mocks/products/productsFixtures.ts

```diff
@@ -16,18 +16,20 @@ import { measureUnitsListFixture } from '../measureUnits/measureUnitsFixture';
 
 export const productsListFixture: Array<ProductServiceResponse> = new Array(130)
   .fill('_')
-  .map((_) => {
+  .map((_, index) => {
     const product: ProductServiceResponse = {
       id: faker.string.nanoid(),
       name: faker.commerce.productName(),
       type: getRandomProperty(ProductServiceTypeEnum),
-      description: faker.datatype.boolean()
-        ? faker.commerce.productDescription()
-        : undefined,
+      description:
+        !index || faker.datatype.boolean()
```

Comment:
Why do we need `index` checks?

Reply:
For the first fixture, you should always call `faker.commerce.productDescription()` since the tests depend on this result (first fixture). Since this pattern is often repeated in our fixtures, I don't consider it a big problem.